logo/README.md

```diff
@@ -0,0 +1,49 @@
+
+This directory contains the different versions of the Slint logo.
+It is (manually) copied into the website under https://slint-ui.com/logo/
+
+## Branding Guidelines
+
+### Logo
+
+ * Do not modify the logo by changing the colors, removing the box, or other alterations to the corporate identity.
+
+ * The logo with the dark box is no longer recommended to use. Prefer always using the logo with the white box.
+
+ * When using the logo to refer to Slint from a location not dedicated to Slint, use the version of the logo with the full tagline
+   ("The fast and easy UI toolkit") `slint-logo-full-light.svg` on light background  or `slint-logo-full-white.svg` on dark background.
+
+   ![Slint logo full](./slint-logo-full-light.svg#gh-light-mode-only)![Slint logo full](./slint-logo-full-white.svg#gh-dark-mode-only)
+
+ * When using the logo to refer to Slint for illustrative purposes, from a place where the viewer know already about Slint
+   (e.g. because it is on the Slint website itself) use the simple version of the logo
+   `slint-logo-simple-light.svg` on light background  or `slint-logo-simple-white.svg` on dark background.
+
+   ![Slint logo simple](./slint-logo-simple-light.svg#gh-light-mode-only)![Slint logo simple](./slint-logo-simple-white.svg#gh-dark-mode-only)
+
+ * For a logo that works on both light and dark background, you can use the square version `slint-logo-square-light.svg`
+
+   ![Slint logo square](./slint-logo-square-light.svg)
+
+ * For an small icon where the text would not be visible otherwise, use `slint-logo-small-light.svg`.
+
+   ![Slint logo small](./slint-logo-small-light.svg)
+
+ * For the rare cases where a circular logo is required, `slint-logo-circle-light.svg` can be used.
+
+   ![Slint logo circle](./slint-logo-circle-light.svg)
+
+### Color
+
+The Slint blue color is #0025FF
+
+### Name
```

Comment:
This section seems like something that belongs to the wiki or so - not sure if the README for the logo is the right place.

Reply:
yeah, i didn't add it a first. But then i thought this is also useful to see for anyone, and here is not a too bad place to put it

Reply:
Alright:). It’s nothing secret after all.